wolfssl/wolfio.h

```diff
@@ -373,6 +373,8 @@
 #endif
 WOLFSSL_API  int wolfIO_TcpConnect(SOCKET_T* sockfd, const char* ip,
     unsigned short port, int to_sec);
+WOLFSSL_API int wolfIO_TcpAccept(SOCKET_T sockfd, SOCKADDR* peer_addr, XSOCKLENT* peer_len);
```

Comment:
The `wolfIO_TcpAccept` needs wrapped in `HAVE_SOCKADDR`

Reply:
Wrappe.